multipart/multipart.py

```diff
@@ -45,6 +45,13 @@
 STATE_PART_DATA_END             = 10
 STATE_END                       = 11
 
+STATES = [
```

Comment:
What's the reasoning for adding this array?  I don't mind, but it doesn't seem to relate to the rest of the PR.

Reply:
I've add another config option to pass delete=False to NamedTemporyFile. Needed to keep the created files without resorting to open-file handle tricks.